mathesar_ui/src/routes/AdminRoute.svelte

```diff
@@ -22,19 +23,23 @@
 
 <Route path="/" redirect={ADMIN_GENERAL_PAGE_URL} />
 
-<LayoutWithHeader>
-  <h1>Administration route</h1>
-  <div>
-    <a href={ADMIN_GENERAL_PAGE_URL}>Goto general</a>
-    <a href={ADMIN_USERS_PAGE_URL}>Goto users</a>
-  </div>
-
-  <Route path="/general">
-    <h2>General</h2>
-  </Route>
+<LayoutWithHeader cssVariables={{ '--max-layout-width': PAGE_MAX_WIDTH }}>
+  <AppSecondaryHeader
+    slot="secondary-header"
+    theme="light"
+    pageTitleAndMetaProps={{
+      name: 'Administrator',
```

Comment:
This should be 'Administration', not 'Administrator'

Reply:
This is fixed in https://github.com/centerofci/mathesar/pull/2423/commits/752484b9913013e47f56660c1bab519488169c2e